app/lib/registries/full_topic_taxonomy_registry.rb

```diff
@@ -37,7 +37,7 @@ def level_one_taxons
 
     def flatten_taxonomy(taxons, output_hash)
       taxons.each do |taxon|
-        output_hash[taxon['base_path']] = format_taxon(taxon)
+        output_hash[taxon['content_id']] = format_taxon(taxon)
```

Comment:
This makes data in the `FullTopicTaxonomyRegistry` a superset of the data in the `TopicTaxonomyRegistry`, which is also keyed by content ID but only has first level taxons.  We should consider removing `TopicTaxonomyRegistry` now.

Reply:
Yep that's a good idea.